Makefile

```diff
@@ -1,5 +1,10 @@
-GO_VERSION=1.9
-GO_FILES=$(shell find . -type f -name "*.go")
+GO_FILES := $(shell find . -type f -name "*.go")
+GO_BUILD := CGO_ENABLED=0 go build -ldflags="-w -s"
+GO_TOOLS := gridx/golang-tools:master-839443d
```

Comment:
This is a certain go version then, right? Is it possible to highlight the used go version?

Reply:
This is not only a certain go version but also a certain version of glide and golint. Yeah, I guess I should somehow reflect it in the tag. I'll open an issue in the grid-x/images repo for this (where the dockerfile is located).